<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>414</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget {
border-radius: 10px;
border-image: url(:/images/reg/bg.jpg);
}

#uppanel {
border-radius:5px;
background: lightgray;
}

#exit {
border-image: url(:/images/icon/close1.png);
}
#exit::hover {
border-image: url(:/images/icon/close2.png);
}
#minimize {
border-image: url(:/images/icon/min1.png);
}
#minimize::hover {
border-image:url(:/images/icon/min2.png);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>261</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>You haven't an account? Create it on the left side</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="log_log">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>169</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_log">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_log">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="log_reg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_reg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_reg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QListWidget" name="uppanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minimize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>exit</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
